src/alexapi/player_state.py

```diff
@@ -0,0 +1,7 @@
+nav_token = ""
```

Comment:
This whole thing should be in `player.py`. The state of the player is a part of it.